.github/workflows/ci_test-full.yml

```
@@ -119,7 +115,7 @@ jobs:
         HOROVOD_BUILT=$(python -c "import horovod.torch; horovod.torch.nccl_built(); print('SUCCESS')" || true)
         if [[ $HOROVOD_BUILT != "SUCCESS" ]]; then
           pip uninstall -y horovod
-          echo $(grep "horovod" requirements/extra.txt) > requirements/horovod.txt
+          echo $(grep "horovod" requirements/strategies.txt) > requirements/horovod.txt
```

nit: this is simpler?
```suggestion
          grep "horovod" requirements/strategies.txt > requirements/horovod.txt
```